src/test/resources/conf/database.properties

```diff
@@ -1,5 +0,0 @@
-databaseConnectionURL=jdbc:mysql://localhost:3306/
```

Comment:
Deleting this file breaks the JUnit Tests. Please unstage this change.

Also please ensure all of the JUnit Tests pass in your local branch before submitting the PR. If they fail there they will definitely fail in Travis